etg/toolbar.py

```diff
@@ -88,14 +100,14 @@ def _fixClientData(c):
 
     # Add some deprecated methods to aid with Classic compatibility.
     # TODO: Which others are commonly enough used that they should be here too?
-    c.addPyMethod('AddSimpleTool', '(self, toolId, bitmap, shortHelpString="", longHelpString="", isToggle=0)',
```

Comment:
The `*SimpleTool` methods are only here for compatibility with Classic, which use the `*HelpString` version of the names (see https://github.com/wxWidgets/wxPython/blob/master/src/_toolbar.i#L217) so I think they should be left alone here.